draft-ietf-httpbis-digest-headers.md

```diff
@@ -250,13 +295,6 @@ Digest: sha-512=WZDPaVn/7XgHaAy8pmojAkGWoRx2UFChF41A2svX+TaPm
                 AbwAgBWnrIiYllu7BNNyealdVLvRwE\nmTHWXvJwew==
 ~~~
 
-Basing `Digest` on representation data takes into account the effect of HTTP
-semantics on messages; for example, the content can be affected by Range
-Requests or methods such as HEAD, while the way the content is transferred "on
-the wire" is dependent on other transformations (e.g. transfer codings for
-HTTP/1.1 - see {{Section 6.1 of HTTP11}}). To help illustrate how such things
-affect `Digest`, several examples are provided in {{resource-representation}}.
-
 A `Digest` field MAY contain multiple representation-data-digest values.
```

Comment:
This `MAY` is redundant, given the abnf rule...

Reply:
agree